python/pyspark/sql/tests.py

```diff
@@ -2819,13 +2802,6 @@ def test_to_pandas(self):
         self.assertEquals(types[4], 'datetime64[ns]')
         self.assertEquals(types[5], 'datetime64[ns]')
 
-    @unittest.skipIf(not _have_old_pandas, "Old Pandas not installed")
-    def test_to_pandas_old(self):
```

Comment:
I have no idea how to test it in jenkins, as jenkins should always have pandas and pyarrow installed with required versions. I think we can only test it manually.

Reply:
jenkins does indeed have pandas and pyarrow installed, btw.